RecoTauTag/Configuration/python/tools/dumpTauVariables_cfg.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("DUMP")
```

Comment:
I think that cfg files are to be more appropriately placed in /test/ subdirectories.